docs/index.html

```diff
@@ -1131,22 +1131,21 @@ <h3>Extensions to Standard</h3>
 
         <section>
           <h4>Verifiable Credential</h4>
-
           <p>The object MUST conform to the basic requirements section of the <a href="https://www.w3.org/TR/vc-data-model/#basic-concepts">W3C VC Data Model</a>.</p>
-
-          <p>The <code>@context</code> MUST contain <code>"https://w3id.org/traceability/v1"</code>.</p>
-
-          <p>The object MUST have an <code>id</code> property, and its value MUST be a valid IRI (URI, URN).</p>
+          <p>It is RECOMMENDED the <code>id</code> be a valid UUID v4 URN, for example <code>urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5</code></p>
```

Comment:
for cases like GS1, we cannot enforce a specific ID value, so I downgrade this to a recommendation